00-projects-template/Makefile

```
@@ -4,6 +4,7 @@
 LIBRARY = 	DS
 TESTERS	=	dll que stk set
 RESULTS	=	results
+INCLUDE	=	-I ../Libft
```

Adding `-I` here could be problematic.

```suggestion
INCLUDE	=	../Libft
```

- Check the `all` rule which uses the `ar` command: `ar` doesn't have an `I` flag.
- Check the `wildcard` function too, the path must be updated.